src/Employer/Employer.Web/ViewModels/VacancyPreviewViewModel.cs

```diff
@@ -1,64 +1,78 @@
-        public string TrainingDescription { get; internal set; }
-        public string TrainingTitle { get; internal set; }
-        public string TrainingType { get; internal set; }
-        public string TrainingLevel { get; internal set; }
-        public string VacancyDescription { get; internal set; }
-        public string VacancyReferenceNumber { get; internal set; }
-        public string WageInfo { get; internal set; }
-        public string WageText { get; internal set; }
-        public string WorkingWeekDescription { get; internal set; }
-
-        public bool HasMapUrl => !string.IsNullOrEmpty(MapUrl);
-
-        public bool HasTrainingProviderDetails => !string.IsNullOrEmpty(ProviderName);
-
-        public bool HasContactDetails =>    !string.IsNullOrEmpty(ContactName) 
-                                            && !string.IsNullOrEmpty(ContactEmail)
-                                            && !string.IsNullOrEmpty(ContactTelephone);
-
+        
+        public VacancyPreviewSectionState ApplicationInstructionsSectionState { get; internal set; }
+        public VacancyPreviewSectionState ApplicationUrlSectionState { get; internal set; }
+        public VacancyPreviewSectionState ContactSectionState { get; internal set; }
+        public VacancyPreviewSectionState ClosingDateSectionState { get; internal set; }
+        public VacancyPreviewSectionState EmployerDescriptionSectionState { get; internal set; }
+        public VacancyPreviewSectionState EmployerNameSectionState { get; internal set; }
+        public VacancyPreviewSectionState EmployerWebsiteUrlSectionState { get; internal set; }
+        public VacancyPreviewSectionState ExpectedDurationSectionState { get; internal set; }
+        public VacancyPreviewSectionState EmployerAddressSectionState { get; internal set; }
+        public VacancyPreviewSectionState NumberOfPositionsSectionState { get; internal set;}
+        public VacancyPreviewSectionState PossibleStartDateSectionState { get; internal set; }
+        public VacancyPreviewSectionState ProviderSectionState { get; internal set; }
+        public VacancyPreviewSectionState QualificationsSectionState { get; internal set; }
+        public VacancyPreviewSectionState ShortDescriptionSectionState { get; internal set; }
+        public VacancyPreviewSectionState SkillsSectionState { get; internal set; }
+        public VacancyPreviewSectionState ThingsToConsiderSectionState { get; internal set; }
+        public VacancyPreviewSectionState TrainingSectionState { get; internal set; }
+        public VacancyPreviewSectionState TrainingLevelSectionState { get; internal set; }
+        public VacancyPreviewSectionState WageTextSectionState { get; internal set; }
+        public VacancyPreviewSectionState DescriptionsSectionState { get; internal set; }
+        public VacancyPreviewSectionState WorkingWeekSectionState { get; internal set; }
+        
+        public bool Wage { get; internal set; }
```

Comment:
Could we call this `HasWage` and `HasProgramme`?

Reply:
These are only used so we have something to map ModelState errors too.